Actions/PrimaryGeneratorAction.h

```diff
@@ -47,19 +52,29 @@ class PrimaryGeneratorAction : public G4VUserPrimaryGeneratorAction
   BeamParams* GetBeamParams() { return fBeam; }
   SourceParams* GetIsotopeParams() { return fIsotope; }
   G4String GetSourceTypeInfo() { return fGenerateSourceType; }
-  void SetNemaPoint(G4int i) { fNemaPoint = i; }
-  G4int GetNemaPoint() { return fNemaPoint; }
+  void SetNemaPoint(G4int pointID) { fNemaGenerator.SetOnePointOnly(pointID); fNemaPoint = pointID; };
+  void GenerateDefaultNemaPositions();
+  void SetNemaPointPosition(int nemaPoint, const G4ThreeVector& position);
+  void SetNemaPositionWeight(int position, int weight);
+  void SetNemaPointLifetime(int position, double lifetime);
+  void SetNemaPoint3GOption(int position);
+  void SetNemaPointSize(int position, G4double radius, G4double length);
+  void SetNemaPointOrientation(int position, double theta, double phi);
+  void SetNemaPointShape(int position, Dimension dim, double G4double, double power, double length);
```

Comment:
"double G4double" looks like an error. Isn't `G4double` a type?
There is also a mixture of `double` and `G4double` in the methods' parameters.